query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/ids/uuid_create_graphql.rs

```diff
@@ -130,4 +130,62 @@ mod uuid_create_graphql {
 
         Ok(())
     }
+
+    fn schema_cuid_2() -> String {
+        let schema = indoc! {
+            r#"model Todo {
+              #id(id, String, @id, @default(cuid(2)))
+              title String
+            }"#
+        };
+
+        schema.to_owned()
+    }
+
+    // "Creating an item with an id field of model CUIDv2 and retrieving it" should "work"
+    #[connector_test(schema(schema_cuid_2))]
+    async fn create_cuid_v2_and_retrieve_it_should_work(runner: Runner) -> TestResult<()> {
+        let res = run_query_json!(
+            &runner,
+            r#"mutation {
+          createOneTodo(data: { title: "the title" }){
+            id
+          }
+        }"#
+        );
+
+        let cuid = match &res["data"]["createOneTodo"]["id"] {
+            serde_json::Value::String(str) => str,
+            _ => unreachable!(),
+        };
```

Comment:
```suggestion
        let cuid = res["data"]["createOneTodo"]["id"].as_str().unwrap();
```

Reply:
Again, this was just copied from the test above